samples/Actor/DemoActor/DemoActor.cs

```diff
@@ -66,13 +66,12 @@ public Task UnregisterReminder()
             return this.UnregisterReminderAsync("TestReminder");
         }
 
-        public Task ReceiveReminderAsync(string reminderName, byte[] state, TimeSpan dueTime, TimeSpan period)
+        public async Task ReceiveReminderAsync(string reminderName, byte[] state, TimeSpan dueTime, TimeSpan period)
         {
             // This method is invoked when an actor reminder is fired.
-            var actorState = this.StateManager.GetStateAsync<MyData>(StateName).GetAwaiter().GetResult();
-            actorState.PropertyB = $"Reminder triggered at '{DateTime.Now.ToString("yyyy-MM-ddTHH:mm:ss")}'";
-            this.StateManager.SetStateAsync<MyData>(StateName, actorState);
-            return Task.CompletedTask;
+            var actorState = await this.StateManager.GetStateAsync<MyData>(StateName);
+            actorState.PropertyB = $"Reminder triggered at '{DateTime.Now:yyyy-MM-ddTHH:mm:ss}'";
+            await this.StateManager.SetStateAsync<MyData>(StateName, actorState);
```

Comment:
This sample code was **very** incorrect. We should never use `GetAwaiter().GetResult()` in samples.